components/web/spawn-fcgi/Makefile

```diff
@@ -22,31 +22,22 @@
 #
 # Copyright (c) 2011, 2013, Oracle and/or its affiliates. All rights reserved.
 #
+BUILD_BITS=64
 
 include ../../../make-rules/shared-macros.mk
 
 COMPONENT_NAME=		spawn-fcgi
-COMPONENT_VERSION=	1.6.3
+COMPONENT_VERSION=	1.6.4
 COMPONENT_PROJECT_URL=	http://www.lighttpd.net/
 COMPONENT_SRC=		$(COMPONENT_NAME)-$(COMPONENT_VERSION)
 COMPONENT_ARCHIVE=	$(COMPONENT_SRC).tar.gz
-COMPONENT_ARCHIVE_HASH=	\
-    sha256:7507a822995731629149040ee5686279b7582ae46f0296efd64b542ef052a90e
+COMPONENT_ARCHIVE_HASH=	sha256:ab327462cb99894a3699f874425a421d934f957cb24221f00bb888108d9dd09e
 COMPONENT_ARCHIVE_URL=	$(COMPONENT_PROJECT_URL)download/$(COMPONENT_ARCHIVE)
 COMPONENT_BUGDB=	utility/spawn-fcgi
 
```

Comment:
TEST_TARGET: $(NO_TESTS)
needs to be added before the include. If not set explicitly, the BUILD_BITS settings will create automatically a test: $(TEST_32_AND_64) target. If you try to run "gmake test" it will fail.

Reply:
Sorry @AndWac i don't sure to understand what you mean. When i add this:

```
TEST_TARGET= $(NO_TESTS)

include $(WS_MAKE_RULES)/common.mk
```
and then do 'gmake test' i see this:

`There are no tests available at this time.`

However, when i delete:

TEST_TARGET= $(NO_TESTS)

and do 'gmake test' without TEST_TARGET line, i can't see any fails. Could you be so kind to tell me what should i see ?

Reply:
It needs to be
TEST_TARGET: $(NO_TESTS)

TEST_TARGET is a target and thus be terminated with :